Reformat Author page and add examples (#27)
metabrainz/bookbrainz-user-guide: docs/style/entities/author.md

[TOC]

## What is an Author?
An author is someone who writes, translates or provides a story
In BookBrainz the "[entity](../../getting-started.md#what-is-an-entity-in-bookbrainz)" "Author" is basically any person who performs any number of book-related roles: writing, drawing, compiling, commissioning, even people who are [just related to two or more other authors may be eligible](#what-about-people-who-have-not-written-anything)
Generally they will fall into these 3 types:

## Types of Authors:

#### Person

> This is the most common type of Author entity, use for individual people.

Reviewer: ```suggestion
This is the most common type of Author entity, used for individual people.
```

> Before adding a new Author always check if one exists already with that name (also remember to search for any [pen names](#pen-names), aliases, misspellings and so on) before creating a new one.

Reviewer: This has "before adding [...] before creating". What about something like this:

```suggestion
Before adding a new Author, always check if there is already an entry in the database for them (make sure to check not only the main author name but also any [pen names](#pen-names), aliases and common misspellings).
```

Reviewer 2: I agree this is an improvement!

> In general, try to add as much identifying information as possible and also add a [disambiguation](../disambiguation.md). This is necessary if another author with the same name exists, but it is always a good idea to add disambiguations, because it makes it easier for users to choose the right author, and also easier to make sure they have right one."

Reviewer: "easier for users to choose the right author, and also easier to make sure they have right one" seems redundant to me?

```suggestion
In general, try to add as much identifying information as possible and also add a [disambiguation](../disambiguation.md). This is required if another author with the same name exists, but it is always a good idea to add disambiguations, because it makes it easier for users to make sure they are choosing the right author."
```

Try to create short, concise disambiguations.
Examples: "18th century novelist", "19th century Belgian playwright" or "Nigerian Africanfuturist writer"

#### Group

> Use this for groups of Authors, like [collective pseudonyms](https://en.wikipedia.org/wiki/Category:Collective_pseudonyms) (pseudonyms used by multiple individuals), [house pseudonyms](https://en.wikipedia.org/wiki/Pen_name#Collective_names) (owned by a publisher and used by the writers working for them) and similar.

Reviewer: This seems like it's missing at least one example of a publisher-owned name? It could probably replace one of the three very similar current examples.

"and similar" sounds better to me like "and the like" but it's probably fine either way.

Author: Thanks, that does make sense. I added one such house pseudonym

Examples:
[Ellery Queen](https://bookbrainz.org/author/a2532a57-11fa-43d4-aa5d-c2892a2d0320), the group pseudonym of *Frederic Dannay* and *Manfred Bennington Lee* used for their [series of the same name](https://bookbrainz.org/series/a807a7e6-6704-48a1-b53c-fee9860b4f2d).
[James S. A. Corey](https://bookbrainz.org/author/c5c71242-f54c-4327-9510-a81064a9a852) pen name used by *Daniel Abraham* and *Ty Franck* for [*The Expanse* series](https://bookbrainz.org/series/c190a1c3-5391-41da-8473-c2bbec5df9a1).
[Sjöwall/Wahlöö](https://bookbrainz.org/author/b3da6047-0b62-494f-b093-10b3fc20ae7f), a Swedish writing duo that publishes the [Martin Beck series](https://bookbrainz.org/series/f2dbd8ee-ad11-4a36-8a4b-1bd5f8e6808a) as a team

#### Persona
Sometimes Authors create characters and also write under that name, whilst giving this character things like a backstory or have them appear in their work.
This is different from regular pen names in that a Persona is presented as its own thing, not just a character *in* the story, but also as the writer of the work in question. [^1]
Examples:

> The writer [Daniel Handler](https://bookbrainz.org/author/394ac1e3-69e8-4eed-afaa-bb06f877c96e) who writes under the pseudonym *Lemony Snicket*, a character and the supposed author of [*A Series of Unfortunate Events*](https://bookbrainz.org/series/abc21a37-600e-4e4f-9fbc-8839249fcdb9).

Reviewer: An author for Lemony Snicket should then be added and linked from here, shouldn't it?

Also, since this is not just a pen name, should it be used for the author of work relationship? Or should those still be the actual writer behind it? This probably needs to be clarified.

Author: Currently we do not consider personas as separate author entities, but I suppose this has been a point of contention (see https://community.metabrainz.org/t/pen-names-as-aliases-or-as-separate-entries/497086).
However at last decision, we are using author credits to credit an edition using the pen name, and an "author wrote work" relationship between the Work and Author (i.e. the real-person-author, which should have an alias added with their various pen names)

Author: We do have a non-deprecated "persona" relationship and even talk about it in this very page, so I might just be misremembering or previously misunderstood.

Author: Ping @ApeKattQuest-MonkeyPython I guess it's a good time to finish defining the "persona" guidelines.
If we don't want a separate Author entity for the Persona (?) I can deprecate the relationship and make a list of where it is used.

Reviewer 2: from what I understand, it was the "is a pen name" that is depreciated and should not be used, while the "persona" is like, if you will, "performance name" of an author. They have a character that is their own whole thing, more than just an alias or pseudonym. 
Myself I haven't come across the "persona" thing in the wild so I can't say that I know how to feel abt the differences between "persona" and "pen-name / pseudonym" but from what I can discern of the community post this seems to be a good distinction?

Reviewer: For what it's worth, there's cases where I would feel the several-entries-per-person approach makes perfect sense in the same way we use it in MB (different names for different styles). "Iain Banks was a Scottish author, writing mainstream fiction as Iain Banks and science fiction as Iain M. Banks" is a fairly straightforward example.

But if there's a good way to filter author entries per pen name, that might work as well.

Reviewer 2: aw man, having a filtering view on author pages would be so good!
we also do need ot implement tabs so we can have "author" credits on one page and relationships with different roles on another. for example.
I envision a separate "pen names" tab would work well.

#### Pen Names
We do not add pen names as separate authors, instead we use [author credits](#author-credits) to show how an edition was credited with another name. 

> (Note: an illustration would be good here)

Reviewer: Is there meant to be a difference between "guideline" pages and "guides" for BB? I would not expect explanatory illustrations of the basic usage of an author credit to go in the same place as everything else, but that might be my MB separated-docs-for-all brain.

Author: We don't currently have that distinction, so it's one big bag of docs

> Additionally, an alias [link to section about aliases] should also be added to the author entity.

Reviewer: ```suggestion
Additionally, an alias [link to section about aliases] matching the pen name should also be added to the author entity.
```

Examples:

> [Stephen King](https://bookbrainz.org/author/128d9490-ee19-4270-a070-32e0a36847f5) has aliases and is credited as *Richard Bachman* and *Beryl Evans*.

Reviewer: ```suggestion
[Stephen King](https://bookbrainz.org/author/128d9490-ee19-4270-a070-32e0a36847f5) has aliases *Richard Bachman* and *Beryl Evans*, and is credited as such when appropriate.
```

### What about people who have not written anything?

> Someone who never wrote anything, but about whom several biographies have been written, for example, can be added to BookBrainz.

Reviewer: Before that, I think we should have something more introductory. What about something like:

```suggestion
Authors can also be created for people who are relevant to the data, but have not written anything themselves. For example, a person about whom several biographies have been written can be added to BookBrainz and be linked to the biographies with [link to the appropriate relationship doc].
```

Reviewer 2: I like this change!

> Another relevant metric is if they link 2 or more Author entities. [^2]

Reviewer: ```suggestion
Another case where an entry can be added is where it is needed to link 2 or more Author entities (for example, to link a grandparent to a grandchild). [^2]
```

> These non-writer people can be to be added as <u>Authors</u>, despite not having written anything themselves.

Reviewer: I think this line could be dropped if the changes above were made? In any case, "can be to be added" seems wrong.

## Author Credits
This section explains about author credits.

When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)

> To illustrate the difference, see these two Editions which contain the same underlying Work, the novel [The Handmaid’s Tale](https://bookbrainz.org/work/f23d22e8-11ba-440a-a990-075152266c21): this [mass market paperback](https://bookbrainz.org/edition/a70f9eb1-b19b-4467-8f6b-c96c4616d829) published by [Seal Books](https://bookbrainz.org/publisher/e8b15024-7b26-4258-9d32-9367890b8a6d), and this [hardcover edition](https://bookbrainz.org/edition/c5d0a3e0-0eb6-4f07-b9f4-5d49a6819524) published by [Heinemann](https://bookbrainz.org/publisher/62754764-fcbd-469a-8ee1-579ccc7747b4).

Author: This could do with more explanations and less links to unrelated entities, after reading it again.

Reviewer: Eventually you'll want a work page and an edition page probably, or at least one for the two combined - and then I'd link to that from here and drop all of this. The example does illustrate the difference fairly well IMO at least (although the links to the publishers are probably not needed - not sure they hurt either though).

## Author Credits
This section explains about author credits.

> When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)

Reviewer: This seems strange. It says it's important to understand the difference, but it doesn't actually do much about explaining the difference and neither does it link to a better explanation?

> {add an example of small collection of short stories by different author but only one autor name on the cover}

Reviewer: This now exists below.

> BookBrainz is a _relationable database_ that tracks the links between different entities, such as, which <u>Author</u> (entity) actually wrote what <u>Work</u> (entity) and so on. But it also is a database of _credited_ information, that is, who is _credited_ as the author.

Reviewer: Do you mean "relational" database?

Also, not sure you need a comma after "such as" here?

Reviewer 2: > Do you mean "relational" database?

yes, that's a better word, and idk, if the omittance of a comma here is better to you I'm not married ot it. :D

> For the relationable part, we use _relationships_ in the database, typically, an "\[Author\] wrote \[Work\]" relationship.

Reviewer: "relational" part? Also, it's strange that nothing else says anything about the "credited information" part mentioned above, is something missing here?

Reviewer 2: relational yes.

      - This is where Author Credits come in: here the Edition of [*Rogues* ](https://bookbrainz.org/edition/6c051f2b-3d49-4e07-9282-eeb85dd35655) is *credited to* [George R. R. Martin](https://bookbrainz.org/author/8b0b4bfa-16c4-48f1-bd45-b416a3ca420b) & [Gardner Dozois](https://bookbrainz.org/author/b87edc09-87ec-4d92-b589-fec1cea47a63) following the cover.
      - The 6 other authors named on the cover as well as the other 15 authors represented in the book do not appear in the Author Credits.
2. The children's book [*Charlie the Choo-Choo*](https://bookbrainz.org/edition/dd16f9d9-3ee1-436e-a0b2-a216377e93ef) was published by [Stephen King](https://bookbrainz.org/author/128d9490-ee19-4270-a070-32e0a36847f5) under the [pen name](#pen-names) "Beryl Evans".
      - The Edition is *credited to* [Beryl Evans](https://bookbrainz.org/author/128d9490-ee19-4270-a070-32e0a36847f5) (links to the Stephen king entry)
      - The Work entity [*Charlie the Choo-Choo*](https://bookbrainz.org/work/b0b5f6d7-39e7-4515-924b-974277593f54) contained in the Edition has the relationship "was written by [Stephen King]((https://bookbrainz.org/author/128d9490-ee19-4270-a070-32e0a36847f5))"

> ## Current Author-linking relationships are:

Reviewer: Is this supposed to be here to stay or is this a temporary thing until a proper `relationships` page exists? Ideally this would be automatically generated from whatever relationships actually exist in the DB, or it will inevitably become outdated fairly quickly.

Reviewer 2: aw, I personally like the fish example 😂 (and monkey agreed having a made-up example to explain how it works, but real examples for, well, examples, is fine) but...
As for the comment, I belive I talked about this with monkey, but he explained how that's not possible for now? chime in please @MonkeyDo

Author: We do have a page for relationships generated from the DB, but we can add more context and examples here.
https://bookbrainz.org/relationship-types/author-work

I do agree this also creates issues with keeping it up to date. Perhaps we should be working on improving the existing page and descriptions in the DB (which we can still do based on this work afterwards), but it removes some flexibility

Reviewer: In MB we have both "description" for rels and a potentially longer section called "Documentation" in the backend but (since that's what we effectively use it for) "Guidelines" in the front, see https://musicbrainz.org/relationship/5be4c609-9afa-4ea0-910b-12ffb71e3821 for an example. That way those are also stored in the DB and easy to access if you want to, for example, have a way to toggle them while editing so the editors can make sure they're doing things right.

Author: Is this just a markdown text field in the DB?

Reviewer: Raw HTML in our case cuz we oldddd, although markdown would probably make sense.

Reviewer 2: ok one question before I give my  👍, are we definitely 100% unwanting  the fish example?
if you both would rather not I will relent to popular opinion 😂*
_(personally I think having a bit fun/silly makes the docs more fun and engaging to read but if it seems dumb or confusing I say strike it.)_

      * **persona**
         (see [above](#persona)) links an Author to their persona.
      * **member of group**
         * **founder**
            (link a person as founder of a group of authors)
         * **subgroup**
            (links a collective as a sub-group to another collective)
      * **marriage**
         (link two authors as married, Note: [non-authors might be eligible here](#what-about-people-who-have-not-written-anything))
      * **involved with**
         (link two authors as involved, see above note)
      * **parent**
         (link an author as being the parent to another, see above note)
      * **sibling relations**
         (link two (or more) authors as being siblings. Take care to link everyone to everyone else!, see above note)
      * **obsolete relationships**, if you see any author with these you are encouraged to fix them!:
         * <del>collaboration</del>
            obsolete, we encourage people to fix old relationships and to use [author credits](#author-credits) and to create collectives (groups) for more permanent collaborations instead.
         * <del>pen name</del>
            obsolete, (see [above](#persona))
2. **Works**

>    Works are the conceptional part of writing! (See [about Works](work.md) for more information)

Reviewer: "conceptual"?

Reviewer 2: yes, please replace that with this better word
